bigbluebutton-tests/playwright/core/elements.ts

```
@@ -282,6 +285,8 @@ export const elements = {
   currentSlideText: 'span[id="currentSlideText"]',
   notesOptions: 'button[data-test="notesOptionsMenu"]',
   exportNotesAsPDF: '[data-test="exportNotesAsPDF"]',
+  blockNoteEditor: 'div[data-test="notes"] .bn-editor',
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)